liquibase-core/src/main/java/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParser.java

```diff
@@ -377,6 +378,10 @@ private void handleProperty(ChangeLogParameters changeLogParameters, DatabaseCha
         changeLogParameters.set(name, value, context, labels, dbms, global, changeLog);
     }
 
+    protected boolean supportExtension(String changelogFile){
```

Comment:
```suggestion
    protected boolean supportsExtension(String changelogFile){
```

Reply:
I think that seems like a better wording. I committed the suggestion